release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -94,6 +94,17 @@ With this enhancement, users can set the `--import-mode` flag to `Legacy` or `Pr
 [id="ocp-4-13-machine-api"]
 === Machine API
 
+[id="ocp-4-13-mapi-cpms-platform-support"]
+==== Additional platform support for control plane machine sets
```

Comment:
This says additional platform support because I plan to stick GCP under here as well.